Resources/config/security.xml

```diff
@@ -9,8 +9,7 @@
         </service>
         <service id="sonata.media.security.superadmin_strategy" class="Sonata\MediaBundle\Security\RolesDownloadStrategy">
             <argument type="service" id="translator"/>
-            <argument/>
-            <!-- Either security.context or security.authorization_checker -->
+            <argument type="service" id="security.authorization_checker"/>
```

Comment:
hmm, as this is master and we already dropped the support, SF 2.8 is the lowest version, so replacing the argument is not needed anymore IMO

The fix of @jenyak looks correct to me @greg0ire